packages/@lightningjs/ui-components/src/components/Column/Column.d.ts

```diff
@@ -18,32 +18,56 @@
 
 import lng from '@lightningjs/core';
 import NavigationManager from '../NavigationManager';
-import type { StylePartial } from '../../types/lui';
-
-type TransitionObject = {
-  delay: number;
-  duration: number;
-  timingFunction: string;
-};
-
-export type ColumnStyle = {
-  itemSpacing: number;
-  scrollIndex: number;
-  alwaysScroll: boolean;
-  neverScroll: boolean;
-  itemTransition: TransitionObject;
-};
-
-export default class Column extends NavigationManager {
+
+declare namespace Column {
+  // errored until NavigationManager TS updates are merged
+  export interface TemplateSpec extends NavigationManager.TemplateSpec {
+    /**
+     * When navigation between multiple Columns,
+     * plinko functionality enables navigation to the item in the next Column that is closest to the index of the item in the previous Column.
+     */
+    plinko?: boolean;
+  }
+}
+
+declare class Column<
+  TemplateSpec extends Column.TemplateSpec = Column.TemplateSpec
+> extends NavigationManager<TemplateSpec> {
+  // Properties
+  /**
+   * When navigation between multiple Columns,
+   * plinko functionality enables navigation to the item in the next Column that is closest to the index of the item in the previous Column.
+   */
   plinko?: boolean;
-  itemPosX?: number;
-  itemPosY?: number;
-  get style(): ColumnStyle;
-  set style(v: StylePartial<ColumnStyle>);
 
+  // Methods
+  /**
+   * Returns the item right before the item that has skipPlinko or before prev if no item has skipPlinko
+   * @param prev component existing right before the current component
+   * @param next component existing right after the current component
+   */
   checkSkipPlinko(prev: lng.Component, next: lng.Component): lng.Component;
+
+  // TODO: flag for discussion
+  // in the docs, there is a parameter for this function but in Column.js the function has no params
+  /**
+   * A callback that can be overridden to do something with the items that are currently on screen.
+   * This will be called on every new render.
+   */
   onScreenEffect(): void;
 
+  // TODO: flag for conversation/discussion
+  // should we include these methods?
+  /**
+   * Removes the passed in item from the items array and updates the selectedIndex, if necessary
+   * @param item component to be removed
+   */
   $removeItem(item: lng.Component): void;
```

Comment:
@erautenberg spoke with Anthony on this but we wanted to confirm with you if we should include these event methods in the TS files?

Reply:
From standup this morning, confirmed these should be included if it's useful for the user to access.